pulp_container/app/tasks/sync_stages.py

```diff
@@ -147,13 +147,14 @@ async def run(self):
                 content_data, raw_text_data, response = await artifact
 
                 digest = calculate_digest(raw_text_data)
+                tag_name = response.url.split("/")[-1]
```

Comment:
Was this change necessary? It looks like the variable is only used on line 172. The code tends to be a lot more readable if you keep the declaration and the usage of the variable close together.